src/Entity/Searches.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\SearchesRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: SearchesRepository::class)]
+class Searches
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
```

Comment:
in my opinion, the fields in this table should be "not null"

Reply:
> in my opinion, the fields in this table should be "not null"

as for me, these validations should be made on the "insert side" (before insert queries): "id" will not be null in any case (its AI), "word" should be checked before insert, "cnt" also should be set before inserting. I do not see any reasons to add one more "layer" of protections for this table but can add it if someone else does not agree with me.